linux/LinuxProcess.h

```diff
@@ -118,6 +119,8 @@ typedef struct LinuxProcess_ {
    /* Autogroup scheduling (CFS) information */
    long int autogroup_id;
    int autogroup_nice;
+
+   char *syscall;
```

Comment:
This needs to be free'd in `linux/LinuxProcess.c:Process_delete()`.

Also please align pointers to the left.